### PR TITLE
docs: add missing point

### PR DESCRIPTION
**Pull request by schmelto**

### CODE REVIEW

**Comment by FreHu** [comment]
I think the reason this can't be merged is because you didn't check the "allow edits from maintainers" option, so I don't have "update from main" available to sync it before merging. Please check the box or update from main yourself, I think you should see the button as the author.

**Review by FreHu — CHANGES_REQUESTED** [review]
see comment above

**Comment by schmelto** [comment]
I`ve opened a new PR for this #4769

**Pull request closed without merging**
